### ISSUE TITLE
Installing arrow on ubuntu 20.04 is broken

### ISSUE THREAD

**Issue by EmmN** [apache/arrow#34680]
### Describe the bug, including details regarding any error messages, version, and platform.

I tried installing using instructions from this page https://arrow.apache.org/install/

```
sudo apt update
sudo apt install -y -V ca-certificates lsb-release wget
wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
sudo apt install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
```

And I get an error on the install.
```
 wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
--2023-03-22 09:28:18--  https://apache.jfrog.io/artifactory/arrow/ubuntu/apache-arrow-apt-source-latest-focal.deb
Resolving apache.jfrog.io (apache.jfrog.io)... 34.231.60.147, 34.225.25.188, 34.230.238.5
Connecting to apache.jfrog.io (apache.jfrog.io)|34.231.60.147|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://landing.jfrog.com/reactivate-server/apache [following]
--2023-03-22 09:28:19--  https://landing.jfrog.com/reactivate-server/apache
Resolving landing.jfrog.com (landing.jfrog.com)... 34.230.238.5, 34.231.60.147, 34.225.25.188
Connecting to landing.jfrog.com (landing.jfrog.com)|34.230.238.5|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 10612 (10K) [text/html]
Saving to: ‘apache-arrow-apt-source-latest-focal.deb’

apache-arrow-apt-source-latest-focal.deb     100%[=============================================================================================>]  10.36K  --.-KB/s    in 0.001s

2023-03-22 09:28:19 (16.6 MB/s) - ‘apache-arrow-apt-source-latest-focal.deb’ saved [10612/10612]


sudo apt install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
Reading package lists... Error!
E: Invalid archive signature
E: Internal error, could not locate member control.tar.{zstlz4gzxzbz2lzma}
E: Could not read meta data from /home/emanuel/projects/apache-arrow-apt-source-latest-focal.deb
E: The package lists or status file could not be parsed or opened.
```

Everything worked just fine until 8h ago.

This is actually preventing me from deploying an application in production.

### Component(s)

Other

**Comment by assignUser** [comment]
Duplicate of #34675 

Sorry for the inconvenience, the apache jfrog instance is down at the moment. We are waiting on feedback from the ASF infra team.

**Comment by shayonj** [comment]
Similar thing happened here as well it looks like? 
https://github.com/apache/arrow/issues/40744
